docs/cudf/source/user_guide/guide-to-udfs.ipynb

```diff
@@ -377,7 +375,14 @@
     "- `str.isspace`\n",
     "- `==`, `!=`, `>=`, `<=`, `>`, `<` (between two strings)\n",
     "- `len` (e.g. `len(some_string))`\n",
-    "- `__contains__` (e.g, `'abc' in some_string`)"
+    "- `__contains__` (e.g, `'abc' in some_string`)\n",
+    "- `strip`\n",
+    "- `lstrip`\n",
+    "- `rstrip`\n",
+    "- `upper`\n",
+    "- `lower`\n",
+    "- `+` (string concatenation)\n",
```

Comment:
We're mixing operator name styles here. `__contains__` is to `in` as `__add__` is to `+`.

No strong preference but maybe `__contains__` should be `in`, or `+` should be `__add__`. It's also fine to leave as-is.